external-crates/move/crates/move-compiler/rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+# chain_width = 80
+# struct_lit_width = 50
+# struct_variant_width = 50
```

Comment:
Let's maybe set this aside for this PR. I think it would be good to shove into `external_crates` in general?

Reply:
🫡